ifrs17-template/Database/MigrationAndScaffolding/Initial.ipynb

```diff
@@ -160,7 +160,9 @@
                 "\n                Discriminator = table.Column<string>(type: \"nvarchar(max)\", nullable: false),",
                 "\n                LinkedDataNode = table.Column<string>(type: \"nvarchar(max)\", nullable: true),",
                 "\n                ReinsuranceCoverage = table.Column<double>(type: \"float\", nullable: true),",
-                "\n                PremiumAllocation = table.Column<double>(type: \"float\", nullable: true)",
+                "\n                PremiumAllocation = table.Column<double>(type: \"float\", nullable: true),",
+                "\n                CashFlowPeriodicity = table.Column<int>(type: \"int\", nullable: true),",
+                "\n                InterpolationMethod = table.Column<int>(type: \"int\", nullable: true)",
```

Comment:
I would like to double check. The only change in the data model was adding two new columns to the DataNodeParameter table. Is that cotrrect?

Reply:
Hi @andrey-katz-systemorph, yes that is correct 2 new properties were added to the `SingleDataNodeParameter` record

<img width="298" alt="image" src="https://user-images.githubusercontent.com/114566025/232011638-272d6a0a-4bc0-412a-8670-ea2b5b4ee393.png">

in this commit here: 792c292.

Reply:
Thanks!